procedures/release-duty/index.rst

```diff
@@ -65,25 +65,21 @@ Email
 
 As ReleaseDuty you need to *subscribe* to certain mailing lists:
 
- * All types of sign-offs and approvals should go to `release signoff mailing list <https://mail.mozilla.org/listinfo/release-signoff>`__
- * All discussion topics should go to `release drivers mailing list <https://mail.mozilla.org/listinfo/release-drivers>`__
```

Comment:
Similarly with above, url is https://groups.google.com/a/mozilla.org/g/release-drivers